src/fields/field_dropdown.js

```diff
@@ -10,10 +10,14 @@ class FieldDropdown extends Blockly.FieldDropdown {
   showEditor_(event) {
     super.showEditor_(event);
     const sourceBlock = this.getSourceBlock();
+    const style = sourceBlock.style;
```

Comment:
nit: do we have a `getStyle` on blocks? I think we should, so that it's a getter instead of a raw property access.

Reply:
Nope, there's a `getStyleName()` but there isn't a getter for the actual style object. Want me to open a bug in core?

Reply:
Yes please

Reply:
Opened https://github.com/google/blockly/issues/8565